.github/ISSUE_TEMPLATE/zReportIssue.yml

```diff
@@ -0,0 +1,117 @@
+
+        At a minimum, include the following pieces of information:
+        We assume you are using Windows PowerShell, and all example commands given should be run in Windows PowerShell, not in PowerShell Core or CMD.
+
+        * Operating System: To get this information run `[System.Environment]::OSVersion.version.tostring()`
+        * Windows PowerShell Version: Run `$PSVersionTable`
+        * Chocolatey CLI Version: Run `choco --version`
+        * Terminal/Emulator: Some knowledge from you are necessary here. CMD, Windows PowerShell, PowerShell Core, Microsoft Terminal, Cmder, and more are possible known terminals or emulators.
+      value: |
+        * Operating System:
+        * Windows PowerShell version:
+        * Chocolatey CLI Version:
+        * Terminal/Emulator:
+    validations:
+      required: true
+  - type: textarea
+    id: installed-packages
+    attributes:
+      label: Installed Packages
+      description: |
+        Sometimes one or more packages installed may be the cause of the problem. Please include the information from the command `choco list -lo`. If this is part of the installation of Chocolatey CLI or no packages
+        is installed (not even Chocolatey); please add `N/A` instead.
+      render: bash
+    validations:
+      required: true
+  - type: textarea
+    id: output-log
+    attributes:
+      label: Output Log
+      description: |
+        Please include the log given by Chocolatey CLI when running in debugging and verbose mode. You can run Chocolatey CLI in debugging and verbose mode using the flags `--debug --verbose`. Alternatively, you can find the log file in the following default location: `C:\ProgramData
+        \chocolatey\logs\chocolatey.log`.
+        If opening an issue for Chocolatey GUI, please include the information from the Chocolatey GUI log file in: `C:\ProgramData\Chocolatey GUI\Logs`. These files are date based, and you are most likely interested in the latest log file created if the bug happened the last time you launched Chocolatey GUI.
+
+        Before including the output log, make sure of the following:
+        - Make sure there is no sensitive data shared.
+        - We need ALL OUTPUT, not just what you may believe is relevant.
+        - We need ALL OUTPUT (including the configuration information). See https://gist.github.com/ferventcoder/b1300b91c167c8ac8205#file-error-txt-L1-L41 for what we need.
+        - If it is hard to reproduce with debug/verbose, the log file already logs with those parameters. Just grab the relevant section from the log file.
+        - Prefer to link to a gist instead of pasting the entire log in the issue. You can create a new gist here: https://gist.github.com/ (see https://docs.github.com/en/github/writing-on-github/editing-and-sharing-content-with-gists/creating-gists).
```

Comment:
```suggestion
        - Preferably link to a gist instead of pasting the entire log in the issue. You can create a new gist here: https://gist.github.com/ (see https://docs.github.com/en/github/writing-on-github/editing-and-sharing-content-with-gists/creating-gists).
```